components/Api/Stability/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { FormattedMessage } from 'react-intl';
+import styles from './index.module.scss';
+import type { PropsWithChildren } from 'react';
+
+type StabilityProps = PropsWithChildren<{ stability: number }>;
+
+const getStabilityClass = (stability: number) => {
+  switch (stability) {
+    case 0:
+      return styles.stabilityLevel0;
+    case 1:
+      return styles.stabilityLevel1;
+    case 2:
+      return styles.stabilityLevel2;
+    case 3:
+      return styles.stabilityLevel3;
+    default:
+      throw new Error(`Unknown stability level: ${stability}`);
+  }
+};
+
+const Stability: React.FC<StabilityProps> = ({ stability, children }) => (
```

Comment:
You need to import the FC types

```suggestion
const Stability: FC<StabilityProps> = ({ stability, children }) => (
```

Reply:
yeah sure